cf-custom-resources/lib/alb-rule-priority-generator.js

```diff
@@ -4,8 +4,10 @@
 
 const aws = require("aws-sdk");
 
-// priorityForRootRule is the max priority number that's always set for the listener rule that matches the root path "/"
-const priorityForRootRule = "50000";
+// minPriorityForRootRule is the min priority number for the the root path "/".
+const minPriorityForRootRule = 48000;
+// maxPriorityForRootRule is the max priority number for the the root path "/".
+const maxPriorityForRootRule = 50000;
```

Comment:
i wonder if it would be nice to leave the lowest priority open and set our max to 49999? kinda like how we don't set the highest priority (0)?

Reply:
nvm, might be a breaking change

Reply:
nvm, _might_ be a breaking change